testdata/serviceconfig.yml

```diff
@@ -17,16 +17,12 @@ rules:
         labels:
           source: source.name | "unknown"
           target: target.name | "unknown"
-          service: api.name | "unknown"
```

Comment:
If we remove them here, we should add the updates to the existing dashboard config in this PR as well that pulls them out too.